ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -80,7 +80,12 @@
     private static final By CONDITIONS_LINK = By.cssSelector(".conditions .info-header i.right");
     private static final By ATTACHMENTS_LINK = By.cssSelector("a[id='attachments.attachments.visitActions.default']");
     private static final By ACTIVE_VISIT = By.cssSelector("visitbyencountertype ul li a");
-
+    private static final By DELETE_PATIENT = By.id("org.openmrs.module.coreapps.deletePatient");
```

Comment:
Delete patient what? Icon, button, link, or something else?

Reply:
ohh sure Delete_Patient_Link looks better , Thanks for the catch.